packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -1117,23 +1121,24 @@ describe('Table Interactive Column Sizing', () => {
     });
 
     it('resizing columns emits single "column-configuration-change" event with expected state', async () => {
-        const listener = createEventListener(
+        const spy = jasmine.createSpy<TableColumnConfigurationChangeEventHandler>();
+        const listener = waitForEvent(
             element,
-            'column-configuration-change'
+            'column-configuration-change',
+            spy
         );
         pageObject.dragSizeColumnByRightDivider(2, [1, 1, 1, 1]);
         await waitForUpdatesAsync();
-
-        expect(listener.spy).toHaveBeenCalledTimes(1);
+        await listener;
```

Comment:
Some examples like this could have been racy as they were not explicitly waiting on the listener. This becomes a lot more evident as the waitForEvent returns the promise directly and we have linting for avoiding floating promises.